src/tests/work-item/work-item-list/quickadd-workitem.spec.js

```diff
@@ -39,18 +38,17 @@ describe('Work item list', function () {
   });
 
   it('Creating a new quick add work item and Cancel delete - phone.', function () {
-    setBrowserMode('phone');
```

Comment:
No, because for all the cases we are using same screen configuration which is already defined in before each part. We would need that if we wanted to change the screen size something other than what we defined in before each.